tasks/main.yml

```diff
@@ -1,4 +1,17 @@
 ---
+
+# If we set these as facts, they are reachable via hostvars
+- set_fact:
+    openvpn_intra_if: "{{ ansible_default_ipv4.interface }}"
+  when: openvpn_intra_if is not defined
+- set_fact:
+    openvpn_intra_ipv4: "{{ hostvars[inventory_hostname]['ansible_' + openvpn_intra_if]['ipv4'] }}"
```

Comment:
The line above returns a list of dicts, each containing `address`, `broadcast`, `netmask`, and `network`…

Reply:
Actually this only happens on my macOS machine. On Linux the value of `ipv4` is always a dict, even if multiple IP addresses are configured.